tools/pd-ctl/pdctl/command/store_command.go

```diff
@@ -31,14 +31,15 @@ var (
 // NewStoreCommand return a stores subcommand of rootCmd
 func NewStoreCommand() *cobra.Command {
 	s := &cobra.Command{
-		Use:   `store [delete|label|weight|limit] <store_id> [--jq="<query string>"]`,
-		Short: "show the store status",
+		Use:   `store [command] [flags]`,
+		Short: "manipulate stores",
```

Comment:
There are readonly commands, `store` or `store 1` do not manipulate stores.

Reply:
How about `manipulate or query stores`?

Reply:
Good idea.